test/api_test.py

```diff
@@ -19,11 +19,18 @@
 import logging
 import unittest
 
+import googleapiclient.errors
+import mock
+import nose.tools
+
 import hyou.api
 
 import http_mocks
```

Comment:
No, I fully agree. The situation is a bit special here though as `hyou` and `http_mocks` are neither standard library nor third party, so I just left it as it was before. As an alternative, what do you think of this ordering
````
from __future__ import (
    absolute_import, division, print_function, unicode_literals)

import contextlib
import logging
import unittest

import googleapiclient.errors
import hyou.api
import mock
import nose.tools

import http_mocks
````
arguing that from the perspective of the tests, `hyou` *is* a third-party library.